rdrf/rdrf/forms/navigation/quick_links.py

```diff
@@ -18,297 +18,257 @@ class QuickLink(object):
-        Users = QuickLink(reverse("admin:groups_customuser_changelist"), _('Users'))
-
-    if settings.SYSTEM_ROLE in (SystemRoles.NORMAL, SystemRoles.CIC_DEV, SystemRoles.CIC_CLINICAL):
-        # related links are grouped or convenience
-        AUDITING = {
-            LoginLog.text: LoginLog,
-            FailedLoginLog.text: FailedLoginLog,
-            LoginAttempts.text: LoginAttempts
-        }
-        DATA_ENTRY = {
-            PatientsListing.text: PatientsListing,
-        }
-        EMAIL = {
-            EmailNotification.text: EmailNotification,
-            EmailTemplate.text: EmailTemplate,
-            EmailNotificationHistory.text: EmailNotificationHistory,
-        }
-        GENETIC = {
-            Genes.text: Genes,
-            Laboratories.text: Laboratories,
-        }
-        IP_RESTRICT = {
-            IPRestrictRangeGroup.text: IPRestrictRangeGroup,
-            IPRestrictGeoGroup.text: IPRestrictGeoGroup,
-            IPRestrictRule.text: IPRestrictRule
-        }
-        if settings.DESIGN_MODE:
-            OTHER = {
-                Sites.text: Sites,
-                Groups.text: Groups,
-                Importer.text: Importer,
-                DemographicsFields.text: DemographicsFields,
-                NextOfKinRelationship.text: NextOfKinRelationship,
-                ArchivedPatients.text: ArchivedPatients,
-                PatientStages.text: PatientStages,
-                ConsentRules.text: ConsentRules,
-            }
-        else:
-            OTHER = {
-                Sites.text: Sites,
-                Groups.text: Groups,
-                DemographicsFields.text: DemographicsFields,
-                Importer.text: Importer,
-                NextOfKinRelationship.text: NextOfKinRelationship,
-                ArchivedPatients.text: ArchivedPatients,
-                ConsentRules.text: ConsentRules,
-            }
-        EXPLORER = {
-            Explorer.text: Explorer,
-        }
-        if settings.DESIGN_MODE:
-            REGISTRY_DESIGN = {
-                Registries.text: Registries,
-                RegistryForms.text: RegistryForms,
-                Sections.text: Sections,
-                DataElements.text: DataElements,
-                CdePolicy.text: CdePolicy,
-                PermissibleValueGroups.text: PermissibleValueGroups,
-                PermissibleValues.text: PermissibleValues,
-                ConsentSections.text: ConsentSections,
-                ConsentValues.text: ConsentValues,
-                ContextFormGroups.text: ContextFormGroups,
-            }
-        REPORTING = {
-            Reports.text: Reports,
-        }
-        STATE_MANAGEMENT = {
-            States.text: States
-        }
-        WORKING_GROUPS = {
-            WorkingGroups.text: WorkingGroups,
-        }
-        # When enabled, doctors links
-        ENABLED_DOCTORS = {
-            Doctors.text: Doctors,
-        }
+def is_clinical_or_dev():
```

Comment:
Wouldn't these `is_SOME_ROLE method` would be better placed on SystemRole.

Additionally, it would make sense to have some `is_CIC` method for all the CIC roles and then we would have `is_proms` (as declared below) and `is_clinical_or_dev` would be CIC but not proms.

Reply:
Updated.